pymongo/monitoring.py

```diff
@@ -635,25 +645,24 @@ def __init__(
         # Command name must be first key.
         command_name = next(iter(command))
         super().__init__(
-            command_name, request_id, connection_id, operation_id, service_id=service_id
+            command_name,
+            request_id,
+            connection_id,
+            operation_id,
+            service_id=service_id,
+            database_name=database_name,
         )
         cmd_name = command_name.lower()
         if cmd_name in _SENSITIVE_COMMANDS or _is_speculative_authenticate(cmd_name, command):
             self.__cmd: _DocumentOut = {}
         else:
             self.__cmd = command
-        self.__db = database_name
 
     @property
     def command(self) -> _DocumentOut:
         """The command document."""
         return self.__cmd
 
-    @property
-    def database_name(self) -> str:
-        """The name of the database this command was run against."""
-        return self.__db
```

Comment:
I think we still need this? Otherwise the docs will say that CommandStartedEvent.database_name was added in 4.6 when it really has existed the whole time.